<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>870</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Reco Workbench</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/assets/icons/ball.ico</normaloff>:/assets/icons/ball.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="reco::datapipe::multichannel_viewer" name="stereo_feed_viewer" native="true"/>
    </item>
    <item>
     <widget class="reco::datapipe::multichannel_viewer" name="disparity_viewer" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="block_size_horizontal_layout">
        <item>
         <widget class="QLabel" name="block_size_label">
          <property name="text">
           <string>block size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="block_size_spin_box">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="block_size_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="minimum_disparity_horizontal_layout">
        <item>
         <widget class="QLabel" name="minimum_disparity_label">
          <property name="text">
           <string>minimum disparity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minimum_disparity_spin_box">
          <property name="maximum">
           <number>256</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="minimum_disparity_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="number_of_disparities_horizontal_layout">
        <item>
         <widget class="QLabel" name="number_of_disparities_label">
          <property name="text">
           <string>number of disparities</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="number_of_disparities_spin_box">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
          <property name="singleStep">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="number_of_disparities_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="singleStep">
         <number>16</number>
        </property>
        <property name="pageStep">
         <number>32</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="speckle_range_horizontal_layout">
        <item>
         <widget class="QLabel" name="speckle_range_label">
          <property name="text">
           <string>speckle range</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="speckle_range_spin_box">
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="speckle_range_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="speckle_window_size_horizontal_layout">
        <item>
         <widget class="QLabel" name="speckle_window_size_label">
          <property name="text">
           <string>speckle window size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="speckle_window_size_spin_box">
          <property name="maximum">
           <number>300</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="speckle_window_size_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="v_offset_horizontal_layout">
        <item>
         <widget class="QLabel" name="v_offset_label">
          <property name="text">
           <string>vertical offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="v_offset_spin_box">
          <property name="minimum">
           <number>-128</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="v_offset_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>-128</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="p1_horizontal_layout">
        <item>
         <widget class="QLabel" name="p1_label">
          <property name="text">
           <string>P1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="p1_spin_box">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>2904</number>
          </property>
          <property name="singleStep">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="p1_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>2904</number>
        </property>
        <property name="singleStep">
         <number>24</number>
        </property>
        <property name="pageStep">
         <number>960</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="p2_horizontal_layout">
        <item>
         <widget class="QLabel" name="p2_label">
          <property name="text">
           <string>P2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="p2_spin_box">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>11616</number>
          </property>
          <property name="singleStep">
           <number>12</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="p2_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>11616</number>
        </property>
        <property name="singleStep">
         <number>24</number>
        </property>
        <property name="pageStep">
         <number>960</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="pre_filter_cap_label">
          <property name="text">
           <string>pre-filter cap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pre_filter_cap_spin_box">
          <property name="maximum">
           <number>256</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="pre_filter_cap_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="uniqueness_ration_horizontal_layout">
        <item>
         <widget class="QLabel" name="uniqueness_rato_label">
          <property name="text">
           <string>uniqueness ratio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="uniqueness_ratio_spin_box">
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="uniqueness_ratio_slider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rectification_horizontal_layout">
        <item>
         <widget class="QCheckBox" name="rectify_checkbox">
          <property name="text">
           <string>Rectify</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="save_current_button">
        <property name="text">
         <string>Save matcher input image pair</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="capture_button">
        <property name="toolTip">
         <string>Start playback &amp; capture of the input video</string>
        </property>
        <property name="text">
         <string>Play / Capture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause_button">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Stereo_Stream">
     <property name="title">
      <string>&amp;Open Stereo Stream</string>
     </property>
     <addaction name="action_open_video_files"/>
     <addaction name="action_wifi_camera"/>
    </widget>
    <addaction name="menuOpen_Stereo_Stream"/>
    <addaction name="action_open_image_pair"/>
    <addaction name="action_open_calibration_file"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="action_open_video_files">
   <property name="text">
    <string>&amp;Video Files</string>
   </property>
  </action>
  <action name="action_wifi_camera">
   <property name="text">
    <string>&amp;Wifi Cameras</string>
   </property>
  </action>
  <action name="action_open_image_pair">
   <property name="text">
    <string>Open &amp;Image Pair</string>
   </property>
  </action>
  <action name="action_open_calibration_file">
   <property name="text">
    <string>Open &amp;Calibration File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>reco::datapipe::multichannel_viewer</class>
   <extends>QWidget</extends>
   <header location="global">reco/datapipe/multichannel_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
